taken from upstream

From efb04936b8ad3bf313c289a522ae5832e22c1b7e Mon Sep 17 00:00:00 2001
From: Paul Newall <quandry@ntlworld.com>
Date: Tue, 15 Oct 2013 22:20:15 +0100
Subject: [PATCH] detection of cups commented out in configure.in it is no
 longer used by kodakaio.

---
 ChangeLog    |  4 ++++
 configure.in | 24 ++++++++++++------------
 2 files changed, 16 insertions(+), 12 deletions(-)

2013-10-15 Paul Newall <quandry@ntlworld.com>
	* configue.in: detection of cups was commented out. Avahi is now used
	for autodetection in kodakaio so the detection of cups is no longer required.

diff --git a/configure.in b/configure.in
index e13888c..4365868 100644
--- a/configure.in
+++ b/configure.in
@@ -332,17 +332,17 @@ if test -c /dev/urandom ; then
     AC_DEFINE(HAVE_DEV_URANDOM, 1, [Is /dev/urandom available?])
 fi
 
-dnl added by PN 3/2/12 to detect cups
-$as_echo "checking for cups"
-if test -e /usr/include/cups/cups.h ; then
-    AC_DEFINE(HAVE_CUPS, 1, [Is /usr/include/cups/cups.h available?])
-	with_cups="yes"
-	LIBS="-lcups  $LIBS"
-else
-	$as_echo "cups.h not found, you may want to install a cups development package"
-	$as_echo "in order to autodetect network scanners in kodakaio."
-	with_cups="no"
-fi
+dnl added by PN 3/2/12 to detect cups commented out 15/10/13 since avahi is now used instead
+dnl $as_echo "checking for cups"
+dnl if test -e /usr/include/cups/cups.h ; then
+dnl     AC_DEFINE(HAVE_CUPS, 1, [Is /usr/include/cups/cups.h available?])
+dnl 	with_cups="yes"
+dnl 	LIBS="-lcups  $LIBS"
+dnl else
+dnl 	$as_echo "cups.h not found, you may want to install a cups development package"
+dnl 	$as_echo "in order to autodetect network scanners in kodakaio."
+dnl 	with_cups="no"
+dnl fi
 
 dnl added by llagendijk 12/7/2012 to detect systemd for saned
 $as_echo_n "Checking for systemd..."
@@ -816,7 +816,7 @@ fi
 echo "IPv6 support:  `eval eval echo ${ipv6}`"
 echo "Avahi support: `eval eval echo ${enable_avahi}`"
 echo "SNMP support:  `eval eval echo ${with_snmp}`"
-echo "CUPS support:  `eval eval echo ${with_cups}`"
+dnl echo "CUPS support:  `eval eval echo ${with_cups}`" commented out 15/10/13
 echo "-> The following backends will be built:"
 for backend in ${BACKENDS} ; do
   echo $ECHO_N "${backend} " 
-- 
2.4.4

